### PR TITLE
add data sources to map

### PR DESCRIPTION
**Pull request by zackarno**
small PR to add data sources for map. Once `map-size-custom` is merged I will change the PR to merge to main instead of `map-size-custom`

### CODE REVIEW

**Comment by caldwellst** [comment]
Okay will hold off on review until `map-size-custom` merged!

**Comment by caldwellst** [comment]
lol simple and easy. The code is obviously fine, I'm just skeptical of adding all of this detail to the email. I think there might need to be a document linked in the email with methods and sources maybe? Since this isn't a one off product, people shouldn't need this information passed in every email iteration, but rather have a place to refer to it. Can see what Leonardo thinks alongside Kevin's comments.

**Review by caldwellst — APPROVED** [review]

**Comment by zackarno** [comment]
Yes, all good points! Since it's internal I'll just merge so that us three can receive this version for a day or 2, consider, then revise as necessary

**Pull request merged**
